index.html

```diff
@@ -48,17 +56,17 @@
             <img src="images\index-images\binabox-cube.webp" alt="binabox cube" class="main-image-cube" width="970"
                 height="846">
             <h1 class="main-heading">WITH <span class="colored">BINABOX</span> EXPLORE NFT COLLECTION</h1>
-            <p>We are the best way to check the rarity of NFT collection</p>
+            <p class="main-sub-text">We are the best way to check the rarity of NFT collection</p>
             <div class="mgroup">
-                <dl>
+                <dl class="list-table">
                     <dt class="list-description">2240+</dt>
                     <dd class="list-term">Total Items</dd>
                     <dt class="list-description">1000+</dt>
                     <dd class="list-term">Profiles Whitelisted</dd>
                 </dl>
             </div>
             <div class="mbutton">
-                <button type="button" class="connect-wallet btn">CONNECT WALLET</button>
+                <button type="button" class="bina-green btn">CONNECT WALLET</button>
                 <button type="button" class="whitelist btn">WHITELIST NOW</button>
             </div>
         </section>

```

Comment:
I noticed that in one of the sections, you have a class `.nft-cards` nested inside another element with the same class `.nft-cards`. It’s better to create distinct class names for these two different entities to avoid confusion and potential styling conflicts.